keras_core/backend/torch/nn.py

```diff
@@ -519,6 +519,9 @@ def one_hot(x, num_classes, axis=-1, dtype="float32"):
     # If axis is not last, change output to axis and shift remaining elements.
     x = convert_to_tensor(x, dtype=torch.long)
 
+    # Torch one_hot does not accept values >= num_classes
+    x = where(x < num_classes, x, 0)
```

Comment:
Please add a unit test to check this case (values >= num_classes)

Reply:
done in b27adc3